docs/source/usage/index.rst

```diff
@@ -225,6 +225,58 @@ In order to use an external provisioning service, simply attach the node to the
 | Attach Network to Node        | ``openstack esi node network attach (--network <network> | --port <port>) <node>`` |
 +-------------------------------+------------------------------------------------------------------------------------+
 
+Serial Console
```

Comment:
Can you name this 'Serial Console Access'? It matches up better with the other section titles.